src/mysqlsh_helpers.py

```diff
@@ -32,6 +55,8 @@ def mysqlsh_bin(self) -> str:
         """
         # Allow for various versions of the mysql-shell snap
         # When we get the alias use /snap/bin/mysqlsh
+        if os.path.exists("/usr/bin/mysqlsh"):
+            return "/usr/bin/mysqlsh"
         if os.path.exists("/snap/bin/mysqlsh"):
             return "/snap/bin/mysqlsh"
         if os.path.exists("/snap/bin/mysql-shell.mysqlsh"):
```

Comment:
Maybe it can be changed to something like the code below (or a lambda function) to avoid repeating the `exist` checks.
```python
for path in ['x', 'y', 'z']:
    if os.path.exists(path)):
        return path
```
But you can ignore it if it makes the code less readable.

Reply:
I like this better also.